Tasks/InstallAppleCertificateV2/Tests/L0CertificateValidForABriefTime.ts

```diff
@@ -37,6 +33,10 @@ const notBefore: Date = new Date(new Date().getTime() - 10 * 60 * 1000);
 // 10 minutes from now.
 const notAfter: Date = new Date(new Date().getTime() + 10 * 60 * 1000);
 
+process.env['AGENT_VERSION'] = '2.116.0';
```

Comment:
These had to moved down since with the new lib overriding TEMP directory in the test doesn't work well since the test runner also overwrites it. The ios signing uses Agent.TempDirectory to put the temp keychain, so the test needs to change it or mock all those methods. This is working, so not investing in mocking all the ios signing methods.

Reply:
I think this is preferable to mocking the ios signing methods, especially since they don't have their own unit tests.

Reply:
I was thinking the same, we lose coverage by mocking them.